templates/volunteer/email/email_application_status_message.txt

```diff
@@ -0,0 +1,6 @@
+{% extends "volunteer/email/base_message.txt" %}
+{% load i18n %}
+
+{% block content %}
+Your current volunteer application status: {{ status }}.
```

Comment:
This is a great start in the right direction.
 I would suggest adding additional information of their application, sometimes it's useful.

Eg, something along the lines of:
"thank you for applying for the {{team names}}. Your application status is {{status}}"

Reply:
Perhaps we can also include link to their profile, saying
"If you would like to review or update your application at any time, go to {{edit_url}}"

Reply:
Thank you for the suggestions!
I have added more information into the template. 

The `edit_url`  uses the `get_absolute_url()`. However, upon clicking the link, it grabs the MailDev's domain instead. 
One way I thought of is prepending `ALLOWED_HOST` to get the correct link when rendering. However, I am unsure if this is a good practice. As such, I did not include it in the PR. 

May I know if the changes are okay?